compiler/erlang_check.erl

```diff
@@ -245,29 +244,26 @@ check_module(File) ->
             report,
             % By adding debug_info, we ensure that the output of xref:m will
             % contain the caller MFAs too.
-            debug_info,
-            {i, AbsDir ++ "/include"},
-            {i, AbsDir ++ "/../include"},
-            {i, AbsDir ++ "/../../include"},
-            {i, AbsDir ++ "/../../../include"}],
```

Comment:
I see your point but I can not justify such things, if this is a real problem I'm sure there are better way of handling it.